src/build.py

```diff
@@ -1508,6 +1548,7 @@ def AllBuilds(use_asm=False):
       Build('llvm', LLVM),
       Build('v8', V8),
       Build('jsc', Jsc, no_windows=True, no_linux=True),
+      Build('jsvu', Jsvu, no_linux=True),
```

Comment:
we can just add no_windows=True here for now too.

Reply:
A Window with no vue? Or a Vue with no window?

Reply:
Ah, so you *don’t* want to install Chakra on Windows yet? Sure, I can add `no_windows=True` to this patch, and we can then remove it in a follow-up commit to add Chakra support.

Reply:
Done. I’ve left the other Windows-specific lines (`if isWindows() …`) intact so we can easily enable support later.

Reply:
I do *want* to install Chakra, it just won't work until someone figures out how to get npm on the bot :) Leaving the windows-specific logic in the function is fine.

Reply:
I’m confused — the bots install a Node.js which comes with npm. See how I set `NPM_BIN` as part of this patch. So, doesn’t this already work?
 https://github.com/WebAssembly/waterfall/pull/325/files#diff-54dfd4a47bc5343af53bfc5a616038f0R209

Reply:
On Windows, the node.js is just bare node.exe without npm (see https://github.com/WebAssembly/waterfall/blob/master/src/build.py#L596). If you download the official zipfile package from Nodejs.org (https://nodejs.org/dist/v8.9.4/node-v8.9.4-win-x64.zip) and attempt to extract it in your waterfall working directory, the extraction will fail because of the ridiculously deep directory structure. Until someone figures out a way to work around that, npm won't work on windows.